requirements.txt

```
@@ -1,5 +1,4 @@
-cython>=0.27,<3
-numpy>=1.20,<2
+numpy>=1.20,<3
```

Based on the discussion in issue ( https://github.com/scikit-learn-contrib/hdbscan/issues/642 ) and looking at commit ( https://github.com/scikit-learn-contrib/hdbscan/commit/142c6ac125b72c8433b7915c1816b5fa0e189122 ), maybe it makes sense to keep the NumPy upper bound at 2?

```suggestion
numpy>=1.20,<2
```

---

Or perhaps I'm mistaken and this works ( https://github.com/scikit-learn-contrib/hdbscan/pull/644 )?

---

Yeah didn't https://github.com/scikit-learn-contrib/hdbscan/pull/644 fix this?